### PR TITLE
docs: add requirements section

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +44,7 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
 
 - [Documentation](#documentation)
```
Comment (JuroUhlar) [inline README.md:45]: Please align the order of sections according to our template: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc 

* TOC
* Requirements
* How to install
* Getting started 
... 
*  Additional resources / Documentation

File: README.md [hunk 2]
```diff
@@ -66,6 +69,16 @@ Using [yarn](https://yarnpkg.com):
...
+
+```sh
+pnpm add @fingerprintjs/fingerprintjs-pro-spa
+```
+
+## Requirements
+
+- For TypeScript users: Typescript 4.5 or higher
+
 ## Getting Started
```
Comment (JuroUhlar) [inline README.md:83]: "Support + Feedback" -> "Support and feedback" 

Consider aligning the copy of this section as well as Licence with our template https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc

File: README.md (hunk 1 above)
Comment (TheUnderScorer) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (TheUnderScorer) [reply README.md:77]: Done!

File: README.md [hunk 3]
```diff
@@ -40,17 +42,16 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
...
+- [Documentation](#documentation)
 - [License](#license)
 
-## Documentation
+## Requirements
 
-This library uses [Fingerprint Pro](https://fingerprint.com/github/) under the hood.
-- To learn more about Fingerprint Pro read our [product documentation](https://dev.fingerprint.com/docs).
-- To learn more about this SDK, there is a [Typedoc-generated SDK Reference](https://fingerprintjs.github.io/fingerprintjs-pro-spa) available.
+- For TypeScript users: Typescript 4.5 or higher
```
Comment (JuroUhlar) [inline README.md:54]: @TheUnderScorer @ilfa Does it make sense to define some kind of [support policy](https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#599e723a44f74dbab6f1ec796a572b19) here or not really?

Comment (TheUnderScorer) [reply README.md:54]: For TypeScript? I'm not sure we need to. @ilfa?

Comment (ilfa) [reply README.md:54]: It looks complicated =( https://github.com/microsoft/TypeScript/blob/main/SUPPORT.md

Comment (TheUnderScorer) [reply README.md:54]: I think that we can omit it in that case because it's not related to the runtime, just the types.

**Review by JuroUhlar — APPROVED** [review]

**Pull request merged**